app/assets/stylesheets/components/header.scss

```diff
@@ -6,7 +6,7 @@
   height: var(--header-height);
   background: var(--header-bg);
   box-shadow: 0 1px 1px var(--header-shadow);
-  z-index: var(--z-sticky);
+  z-index: calc(var(--z-sticky) + 1);
```

Comment:
That's probably not ideal to overwrite that variable since the whole point of z-index cleanups we've done while ago, and overall using variables per se, was to avoid situations like that...

I'd much rather recommend overwriting z-index in this actions panel for appropriate breakpoint: https://github.com/forem/forem/blob/44466532dd84e40c152588be2a08b27507684a2c/app/assets/stylesheets/views/article.scss#L79-L89 - so basically adding `z-index: var(--z-elevate)` inside that breakpoint..

It will keep the code clean without unnecessary calculations and it will still be logical: we set `z-sticky` for smallest breakpoints (which is correct) and once we _relayout_ the site on `$breakpoint-m`, we overwrite `z-sticky` with `z-elevate` since this element no longer needs to be "sticky" (in regards to z-index) but just elevated over other parts of content (so the dropdown render properly)

Reply:
Thank you for your insights into the solution. Really helpful. @ludwiczakpawel